style/style.css

```diff
@@ -90,18 +84,15 @@
   font-size: 20px;
 }
 
-
 .copyright_sign {
-  color: #FFFFFF;
+  color: #ffffff;
   margin: 0 1082px 120px 105px;
```

Comment:
you do not need this huge margin to the right, in fact you just need to write a margin to the left.